spec/models/identity_spec.rb

```diff
@@ -24,6 +24,25 @@
       end
     end
 
+    context "Protocols present for clinical_provider and super_user organizations" do
+
+      it "should return an array of Protocols" do
+        identity            = create(:identity)
```

Comment:
identity_one = create(:identity)
    identity_two = create(:identity)
    protocol_one = create(:protocol, "associate to identity_one)
    protocol_two = create(:protocol, "associate to identity_two)
    etc.
 
    expect(identity_one.protocols.map(&:id)).to eq([protocol_one.id])